tests/test_filters.py

```diff
@@ -95,32 +103,32 @@ def test_remove_tilt_no_mask(test_filters_random: Filters, image_random_remove_x
     np.testing.assert_allclose(tilt_removed, image_random_remove_x_y_tilt, **TOLERANCE)
 
 
-def test_extract_medians(test_filters_random: Filters):
-    """Test extraction of medians"""
-    quantiles = test_filters_random.row_col_quantiles(test_filters_random.pixels, mask=None)
-    row_medians = test_filters_random._extract_medians(quantiles["rows"])
-    target = np.quantile(test_filters_random.pixels, 0.5, axis=1)
+# def test_extract_medians(test_filters_random: Filters):
```

Comment:
We can remove this as the `_extract_medians()` function is no longer in `filters.Filters`.

Reply:
Done 👍